service/run.bash

```diff
@@ -16,6 +16,10 @@
 # Use this script to clone a new copy of fuzzbench and run a requested
 # experiment.
 
+# Append stdout and stderr to /tmp/fuzzbench-service.logs. Redirecting the
+# stdout/stderr of this script in the cron command didn't work for some reason.
+exec &>> /tmp/fuzzbench-service.logs
```

Comment:
I tried this and the most intuitive solutions `&>` and they didn't seem to work. Maybe they didn't work because I tried them while fixing other issues and those other issues prevented this from working. But I can't spend more time on this now, so I think this solution, imperfect as it may be is good enough for now.